### PR TITLE
ruby

### PR DESCRIPTION
**Pull request by solomoncyj**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/58472)

### CODE REVIEW

File: manifests/r/Ruby/Rubydevkit/3.1.2/Ruby.Rubydevkit.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Ruby.Rubydevkit
```
Comment (Trenly) [inline manifests/r/Ruby/Rubydevkit/3.1.2/Ruby.Rubydevkit.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: RubyInstallerTeam.RubyWithDevKit
```

**Review by Trenly — CHANGES_REQUESTED** [review]
Package exists under different Identifier; New versions should use existing ID

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
You are currently removing tons of metadata.

**Comment by Trenly** [comment]
> You are currently removing tons of metadata.

And missing the x86 installer

**Pull request closed without merging**